crates/ruff_server/src/edit/notebook.rs

```diff
@@ -138,11 +138,10 @@ impl NotebookDocument {
                 for cell in structure.array.cells.into_iter().flatten().rev() {
                     if let Some(text_document) = deleted_cells.remove(&cell.document) {
```

Comment:
Nit: We could now move the `sell.insert` call out of the `if else` and instead return the `content` and `version` from the `if else`

```rust
let (content, version) = if let Some(text_document) = deleted_cells.remove(&cell.document) {
	let version = text_document.version();
	(text_document.into_contents(), version)
} else {
	(String::new(), 0)
};
```